src/styles/typography.css

```diff
@@ -3,37 +3,47 @@
 @import url('https://fonts.googleapis.com/css?family=Montserrat:300,400|Open+Sans:400,500,700');
 
 /* Molten leading -  https://css-tricks.com/molten-leading-css/
-   ========================================================================== */
 
-:root {
-  --base-font-size: 14px;
-  // These values are the minimum and maximum viewport sizes to apply the font scaling
-  --min-width: 400;
-  --max-width: 400;
-  // These values represent the range of fon-tsize to apply
-  // These values effect the base font-size, headings and other elements will scale proportionally
-  --min-font: 12;
-  --max-font: 26;
-}
+  This fluidly scales our font-size and line-height/leading as the viewport and line-width 
+  decreases to maximize readablity. 
+
+  Set our base font-size to a minimum of 14px and a max 16px:
+  < 320px (min-width: 20em): 1rem = 14px 
+  320 to 544 px wide: 1rem is calculated as a range between 14px and 16px
+  > 544px (min-widht: 80em): 1rem = 16px
+
+  Since we are scaling the font-size of the html value all other rem values, think heading sizes, margins, padding 
+  will fluidy scale accordingly.  
+  ========================================================================== */
 
 html {
-  font-size: var(--base-font-size);
+  font-size: 14px;
+  line-height: 1.5em;
 }
+
 body {
-  font-size: 1em;
+  font-size: inherit;
   line-height: 1.5em;
 }
 
+/* we use em as a min-width so that it is tied to the line width of typogrpahy 
+and not discrete viewport size. This helps to ensure we only add breakpoints that 
+are related to design decisions we can control and not browser sizes we cannot control */
 @media screen and (min-width: 20em) {
-  body {
-    font-size: calc(var(--base-font-size) + (1.1428571429 - 1) * ((100vw - 20em) / (80 - 20)));
+  /* 
+    1em = 1rem = 14px 
+    thus
+    16px max font-size = 16/1em = 1.1428571429
```

Comment:
This makes more sense

Reply:
more sense than?

Reply:
Where that number came from, it's just 16/14.

Reply:
Just trying to understanding things a bit better... em is a relative font size that requires specifying a frame of reference while px is absolute, right? So a point is 1/72 of a inch, so if you specify em in a 16px font size that's 16/72 and helps specify other aspects around that font, yes/no? If so, I think I still don't get why 14px is used and then why the rest of the font-size calculation and how that relates to the line-height calculator?

Reply:
I don't think it's helpful to think of points or inches here, since it changes drastically based on screen. 
But yeah, `em` is a measure normalized to the container's parent, and since it's being placed on the `<html>` tag, it dictates the entire document's sizing.

The font-size `calc` below actually has a variable `vw` which is the size of the viewport which thus gives us a font size with minimum `14px` that scales linearly to a max of `16px` between the two break points at 80 and 20 (`em` I think?)

The line height `calc` does the same linear scaling, but between `1.5em` and `1.8em` 

So yes, the comment `1em = 1rem = 14px ` is not really true maybe misleading.

Not sure if this clarified anything.

Reply:
Ah, yes it did - thanks.